### ISSUE TITLE
Error during compile if Window.Title is set in XAML.

### ISSUE THREAD

**Issue by lucent-sea** [microsoft/WindowsAppSDK#1722]
### Describe the bug

From a fresh WinUI 3 in Desktop project from Windows App SDK templates, if the Title property in set in MainWindow.xaml, the following error is thrown when compiling:

```
Severity	Code	Description	Project	File	Line	Suppression State
Error	WMC0612	The XAML Binary Format (XBF) generator reported syntax error '0x09C4' : Property Not Found	WinUI	C:\Repos\WinUI\WinUI\WinUI\MainWindow.xaml	8	
```

### Steps to reproduce the bug

- Create a new WinUI 3 in Desktop project from Windows App SDK C# VS2022 Templates.
- Update NuGet packages to Preview 3 versions.
- Open MainWindow.xaml.
- Set Title property to any string on the Window element.
- Try to compile.

### Expected behavior

It should compile with the Title property set.

### IDE

Visual Studio 2022

### NuGet package version

Microsoft.WindowsAppSDK 1.0.0-preview3

### Project type

- [X] Packaged
- [ ] Unpackaged

### Windows version

Windows 11 (22000)

**Comment by martibravo** [comment]
Me too in build 22489. For the record, it is the Xaml Compiler complaining. Can't get the exact wording rn but it was something like "XBF: Property not found".

**Comment by riverar** [comment]
Duplicate of microsoft/microsoft-ui-xaml#3689. This is a known issue and won't be fixed in time for 1.0. Workaround in the aforementioned issue.
